ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/tools/osb/mbean/ObjStoreBrowser.java

```diff
@@ -341,9 +338,23 @@ public ObjStoreBrowser() {
      * In other words, this class should be considered as a singleton class. In fact, if two ObjStoreBrowser instances
      * are created, they may interfere with each other.</p>
      */
-   public ObjStoreBrowser(String logDir) {
-        init(logDir);
-        this.objStoreBrowserMBeanName = arjPropertyManager.getObjectStoreEnvironmentBean().getJmxToolingMBeanName();
+   private ObjStoreBrowser(String logDir) {
+       init(logDir);
+       this.objStoreBrowserMBeanName = arjPropertyManager.getObjectStoreEnvironmentBean().getJmxToolingMBeanName();
+    }
+
+
+    public static ObjStoreBrowser getInstance(String logDir) {
```

Comment:
I am not super -1 on this and not sure if there is a better option but it's a bit - I don't know the right words - strange or unintuitive (perhaps?)  -to see the calls with getInstance(null) used. Like I say I don't know what to suggest but basically once initialized this logDir parameter is going to be ignored...

Reply:
+1, that makes sense to me thank you!